calico-cloud/image-assurance/overview.mdx

```diff
@@ -0,0 +1,37 @@
+---
+description: About Calico Cloud Image Assurance.
```

Comment:
Title should be init caps (aka Overview).

Reply:
Not sure I understand this? Could you explain more ? There is no `Overview` in the code you highlighted